src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -112,12 +150,16 @@ class SideNav extends Component {
         <ul
           id="va-sidenav-ul-container"
           className={classNames(
-            `va-sidenav vads-u-margin-top--0 vads-u-padding--0`,
+            'va-sidenav',
+            'vads-u-margin-top--0',
+            'vads-u-padding--0',
           )}
         >
           <div
             className={classNames(
-              `va-sidenav-display-onclick-line medium-screen:vads-u-display--none line`,
+              'line',
```

Comment:
Hm I don't remember a `line` class

Reply:
It's here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/sass/modules/_m-side-nav.scss#L67